### DISCUSSION TITLE
KubernetesPodOperator pods Evicted

### DISCUSSION THREAD

**Discussion by chuxiangfeng** [apache/airflow#27906] [General]
Using the KubernetesPodOperator runs on k8s, for a shortage of resources is Evicted, but the airflow ui interface shows a task is still running. How can this be recognized, and a task is flagged for failure

**Comment by potiuk** [comment]
I think you forgot to write information about the versions of your airflow, cncf.kubernetes etc. The behaviour might be different in different versions and over the last year there were multiple improvements and fixes.

Not specifying all information makes it difficult to help you with your problem (somehow you make it difficult but not providing the context) - but in any case I'd recommend you to upgrade to latest versions and try there. And if you still see an issue with that - open a a new issue wiht all details and logs - this way you might give us at least a chance to help to solve your problem.
